inc/customizer.php

```diff
@@ -15,6 +15,9 @@ function twentyseventeen_customize_register( $wp_customize ) {
 	$wp_customize->get_setting( 'blogdescription' )->transport  = 'postMessage';
 	$wp_customize->get_setting( 'header_textcolor' )->transport = 'postMessage';
 
+	// Move the Front Page widget area to the Theme Options panel.
+	$wp_customize->get_section( 'sidebar-widgets-front-page' )->panel = 'twentyseventeen_options_panel';
```

Comment:
Huh, I guess the section doesn't always exist when the controls are registered. I didn't spend much time on the code since this is more about exploring the concept, but this should be fixed in cc2b7d6. Thanks for letting me know about that!